app.py

```diff
@@ -42,25 +43,25 @@ def get_dict(self):
 
 class Subjects(db.Model):
 	__tablename__ = 'subjects'
-	id = db.Column(db.Integer, primary_key=True)
+	id = db.Column(db.BigInteger, primary_key=True)
 	subject = db.Column(db.Text, unique=True, nullable=False)
 
 
 class Channels(db.Model):
 	__tablename__ = 'channels'
-	id = db.Column(db.Integer, primary_key=True)
+	id = db.Column(db.BigInteger, primary_key=True)
 	channel = db.Column(db.Text, unique=True, nullable=False)
 
 
 class Posts(db.Model):
 	__tablename__ = 'posts'
-	id = db.Column(db.Integer, primary_key=True)
+	id = db.Column(db.BigInteger, primary_key=True)
 	user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
```

Comment:
foreign keys mut be the same type like their origin columns, Bigint instead of Int. the engine did not raise any error when building the db?